docker/entrypoint_prod.sh

```diff
@@ -2,32 +2,19 @@
 
 # The `release` process type specifies the command to run during deployment, and is where
 # we run DB migrations and other tasks that are 'release' rather than 'build' specific:
-# https://devcenter.heroku.com/articles/release-phase
-# https://devcenter.heroku.com/articles/runtime-principles#build-release-run
 if [ "$1" == "release" ]; then
     exec ./bin/pre_deploy
 
-# The `web` process type is the only one that receives external traffic from Heroku's routers.
 # We set the maximum request duration to 20 seconds, to ensure that poorly performing API
-# queries do not consume a gunicorn worker for unbounded lengths of time. See:
-# https://devcenter.heroku.com/articles/python-gunicorn
-# The Heroku Python buildpack sets some sensible gunicorn defaults via environment variables:
-# https://github.com/heroku/heroku-buildpack-python/blob/master/vendor/python.gunicorn.sh
-# https://github.com/heroku/heroku-buildpack-python/blob/master/vendor/WEB_CONCURRENCY.sh
-# TODO: Experiment with different dyno sizes and gunicorn concurrency/worker types (bug 1175472).
+# queries do not consume a gunicorn worker for unbounded lengths of time. 
```

Comment:
I have no idea if this is still relevant - if our cloudOps deployment has the same concept of a web worker. Interestingly, I do still see new relic sections for web but its unclear to  me if that's from this or something else reporting to it.

Reply:
@oremj please advise here also re web worker.

Reply:
The running docker image still uses gunicorn, so long queries will still consume a worker for as long as they are running and there are a limited number of workers available.